src/components/editor/inputs/InputLang.jsx

```diff
@@ -17,32 +17,46 @@ import { selectLanguages, hasLanguages } from "selectors/languages"
  * See ISO 639 for the list of registered language codes
  */
 const InputLang = (props) => {
-  const [lang, setLang] = useState(props.lang || "")
-  const langPresent = typeof lang !== "undefined"
-  const [languageSelectorEnabled, setLanguageSelectorEnabled] =
-    useState(langPresent)
+  const noLangSelected = "absent" // the values of the radio buttons for when a language is selected or not
+  const langSelected = "present"
+
+  const [lang, setLang] = useState(props.lang)
+  const [submitEnabled, setSubmitEnabled] = useState(true)
+  const [radioButtonValue, setRadioButtonValue] = useState(
+    props.lang === null ? noLangSelected : langSelected
+  )
 
   const classes = ["modal", "fade"]
   let display = "none"
+  const alertClasses = ["alert", "alert-danger"]
```

Comment:
I'd prefer to not use an alert here. It is not consistent with the rest of the app. The Submit button should be disabled when the form is in an invalid state (i.e., when the user has selected to provide a language, but has not done so.).

Reply:
Got it, I'll ditch the alerts but leave the submit button enabling/disabling.